validator/java/OWNERS

```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'nhant01'}, {name: 'GeorgeLuo'}],
```

Comment:
Maybe include `@wg-caching` as a silent owner, and notify @GeorgeLuo and @nhant01 about changes?

Reply:
`wg-caching` owns `validator/` already, so the question is who should be suggested as a reviewer. My inclination would be to add one of these two users as reviewer for changes, and `wg-caching` will still have approval power if they are requested. WDYT?

Reply:
That works 👍